index.html

```diff
@@ -539,6 +539,10 @@ <h2 id="steps">Steps for
 						<span data-profile="CR" data-cr="new">The Group <span class="rfc2119">MUST</span> show that the
 							specification has received
 							wide review in order to move to Candidate Recommendation.</span>
+						<span data-profile="CR" data-cr="new|snapshot">The Group <span class="rfc2119">MUST</span> show that 
+						    all horizontal <code>*-needs-resolution</code> issues have been closed by the relevant 
```

Comment:
I thought about that: wouldn't we want snapshot requests to have at least considered their open horizontal issues? It could be okay to ship with open issues (*cough* HTML *cough*, *cough* CSS *cough*), but I think maybe the gate should be a little bit sticky?